tests/Microsoft.Identity.Test.Integration.netcore/WAM/RuntimeBrokerTests.cs

```diff
@@ -174,47 +174,19 @@ public async Task WamUsernamePasswordRequestAsync()
             Assert.IsNotNull(accounts);
             Assert.AreEqual(0, accounts.Count());
         }
+      
 
         [TestMethod]
-        public async Task WamUsernamePasswordRequestMsaPassthroughAsync()
+        public async Task WamUsernamePasswordRequestOldBrokerAsync()
```

Comment:
Can we move this to be a unit test?

Reply:
Removing this since it is no longer valid as per https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3400#discussion_r908418469.